angular/core/components/thread_page/activity_card/new_comment.scss

```diff
@@ -6,3 +6,36 @@
 .new-comment__in-reply-to {
   a { color: $primary-text-color; }
 }
+
+.new-comment__truncated {
+  max-height: 180px;
+  overflow: hidden;
+  margin-bottom: 0 !important;
+  p { margin-bottom: 0 }
```

Comment:
Because there are margins on the other classes associated with this particular element. They were overriding the css I was trying to add. Is that all good?